modules/rest-api/pages/rest-modify-index-batch-size.adoc

```diff
@@ -0,0 +1,56 @@
+= Modify Index Batch Size
+:description: The REST API supports modification of the batch size whereby the relocation and rebuilding of indexes, during rebalance, is maintained at a high level of performance.
+
+[abstract]
+{description}
+
+
+== HTTP Method and URI
+
+----
+POST /settings
+----
+
+== Description
+
+Establishes the _batch size_, which is used for index rebuilding and metadata transfer, during rebalance.
```

Comment:
Add: "This is the maximum number of indexes that will be moved concurrently in the cluster."

Reply:
Have added.